### ISSUE TITLE
Tick Types Article: Flash missing for Boulder

### ISSUE THREAD

**Issue by rouletout** [theCrag/website#3381]
**What happened?**

Article https://www.thecrag.com/de/artikel/ticktypes does not list Flash as a valid Boulder Tick Type.

<img width="718" alt="screen shot 2019-02-28 at 09 06 53" src="https://user-images.githubusercontent.com/19162952/53575997-5adab200-3b38-11e9-9646-6400935f96e7.png">

Ideally also provide screenshots:

**What you expected:**

Flash as Boulder Tick Type

**Comment by Mdemaillard** [comment]
Merging from duplicate

What happened?
User confused about with ticktypes while bouldering

Example url(s) to reproduce the problem:
https://www.thecrag.com/discussion/3131974173/lead-repeat

What you expected:
https://www.thecrag.com/en/article/ticktypes

onsight and flash should be added to the Bouldering help table, in line with the tick log list
image
image

(Don't move onsight and flash from "Lead" to "General ascent", since the flavor text will differ.)

Proposal for Boulder onsight text:
I completed this boulder cleanly, on my first attempt without prior inspection (touching holds) or beta (Beta is information from someone else. Use of guidebook to locate the route is generally considered OK, but discussing tips for the boulder with someone is not).

Proposal for Boulder Flash text:
I completed this boulder cleanly, on my first attempt, but used prior inspection and/or beta.
